src/App.tsx

```diff
@@ -1,6 +1,68 @@
-import React from 'react';
+/* eslint-disable padding-line-between-statements */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { FC, useState, useCallback } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 
-const App: React.FC = () => <h1>Pagination</h1>;
+const App: FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const [withInfo, setWithInfo] = useState(false);
+  const totalItems = 42;
+  const totalPages = Math.ceil(totalItems / itemsPerPage);
+
+  const onPageChange = useCallback(
+    (page: number) => {
+      setCurrentPage(page);
+    }, [],
+  );
+  return (
+    <div className="App">
+
+      <h1>Pagination</h1>
+
+      <label htmlFor="withInfo">
+        Show additional information:
+        {' '}
+        <input
+          type="checkbox"
+          id="withInfo"
+          checked={withInfo}
+          onChange={(event) => {
+            setWithInfo(event.target.checked);
+          }}
+        />
+      </label>
+
+      <label htmlFor="itemsPerPage">
+        {'Items per page: '}
+        <select
+          id="itemsPerPage"
+          name="itemsPerPage"
+          value={itemsPerPage}
+          onChange={(
+            event => setItemsPerPage(+event.target.value)
+          )}
+        >
+          <option value="3">3</option>
+          <option value="5">5</option>
+          <option value="10">10</option>
+          <option value="20">20</option>
+        </select>
+      </label>
+
+      <Pagination
```

Comment:
add Pagination component to pull request
![image](https://user-images.githubusercontent.com/57574816/171619611-6a8a9ba2-a3b4-46c7-874f-9cc6052f09a2.png)

Reply:
oops ) added )